packages/components/src/Autocomplete/Autocomplete.test.tsx

```diff
@@ -162,3 +162,112 @@ test("it should call the handler skipping headings when an option is selected",
 
   expect(changeHandler).toHaveBeenCalledWith(headingOptions[1].options[0]);
 });
+
```

Comment:
Nice! Tests are so good for us to invest in. They are docs and also examples for other devs to look at when they write their own tests.